scripts/generate_swagger.sh

```diff
@@ -2,6 +2,7 @@
 PATH_TO_PROJECT_ROOT=$(git rev-parse --show-toplevel)
 SWAGGER_DIR="${PATH_TO_PROJECT_ROOT}/pkg/swagger"
 PUBLIC_PATH="${PATH_TO_PROJECT_ROOT}/webui/public/swagger"
+DOCS_PATH="${PATH_TO_PROJECT_ROOT}/docs"
```

Comment:
can we revisit the structure of these directories? I really don't understand why we are involving `webui` in this. I think it was a wrong decision at some point and I would like to see if we can avoid that

Also does it doesn't make sense to have the swagger generation code under `/pkg/swagger` as `/pkg` should only include code related to bacalhau and not for external tooling

Reply:
Makes sense. I am creating an issue to fix it in V2 SDK. #4200 .